compiler/src/iree/compiler/Codegen/Common/test/tile_and_distribute_to_workgroups.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: iree-opt --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-codegen-tile-and-distribute-to-workgroups)), canonicalize, cse)' --split-input-file %s | FileCheck %s
+// RUN: iree-opt --pass-pipeline='builtin.module(hal.executable(hal.executable.variant(iree-codegen-tile-and-distribute-to-workgroups{max-workgroup-parallel-dims=1})), canonicalize, cse)' --split-input-file %s | FileCheck %s -check-prefix=CHECKW
```

Comment:
Just FYI, adding it this way means that all the tests in the file are run once more, but only one of these is checked. I think thats OK, it still shows that these tests run and dont crash with this option which is good.

Reply:
IC makes sense. I originally had a separate file but Thomas suggested they be merged.

Reply:
what would be nice would be to re-use directly an existing test but that's a detail

Reply:
No need to change it. Just pointing out what is happening